echopype/convert/set_groups_ek80.py

```diff
@@ -39,6 +39,31 @@ def set_env(self, env_only=False) -> xr.Dataset:
             if k in dict_env:
                 dict_env["absorption_indicative"] = dict_env.pop(k)
 
+        if "sound_velocity_profile" in self.parser_obj.environment:
+            dict_env["sound_velocity_profile"] = (
+                ["ping_time", "sound_velocity_profile_depth"],
+                [self.parser_obj.environment["sound_velocity_profile"][1::2]],
+                {
+                    "long_name": "sound velocity profile",
+                    "standard_name": "speed_of_sound_in_sea_water",
+                    "units": "m/s",
+                    "valid_min": 0.0,
+                    "comment": "parsed from raw data files as (depth, sound_speed) value pairs",
+                },
+            )
+
+        vars = {
+            "sound_velocity_source": "sound_velocity_source",
+            "transducer_name": "sonar_name",
+            "transducer_sound_speed": "sonar_sound_speed",
```

Comment:
@emiliom : here do you think we should keep the original names `transducer_*` instead of changing to `sonar_*`?  Since it is not entirely clear what this is other than my conjectured use of line of sight time delay to measure sound speed, it seems better to keep its original name. This is not in the convention.

Reply:
The EK80 has a `transducer_sound_speed` value separate from water column sound speed because some of their systems potentially do a correction for transducer parameters based on the sound speed at the transducer face (e.g., beamwidth changes with sound speed across the transducer face).

Reply:
Interesting... Thanks @gavinmacaulay for explaining where this is from!
Is this something that's done automatically based on the transducers? or something the users can toggle on/off?

Reply:
I've never actually seen a Simrad system that uses this, but think that the bathymetric systems may have a use for it.